### PR TITLE
Added Language Contributors

### PR DESCRIPTION
**Pull request by yakov116**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,7 +101,13 @@ You love FastHub? You want new features or bug fixes?
 Please **contribute** to the  project either by [_creating a PR_](https://github.com/k0shk0sh/FastHub/compare) or [_submitting an issue_](https://github.com/k0shk0sh/FastHub/issues/new) on GitHub.  
 Read the [**contribution guide**](.github/CONTRIBUTING.md) for more detailed information.
 
-- [Translation contributors list](.github/language_contributors.md)
+### [Language Contributors](.github/language_contributors.md)
+
```
Comment (k0shk0sh) [inline README.md]: This is a preview test.

File: README.md [hunk 2]
```diff
@@ -101,7 +101,13 @@ You love FastHub? You want new features or bug fixes?
...
+- Turkish [@kutsan](https://github.com/kutsan)
+- German [@failex234](https://github.com/failex234)
+- Chinese [@Devifish](https://github.com/Devifish)
+- Indonesian [@dikiaap](https://github.com/dikiaap)
```
Comment (k0shk0sh) [inline README.md]: This is a second preview test.

Comment (yakov116) [inline README.md]: This is a reply from me!

**Comment by k0shk0sh** [comment]
Hello world

File: README.md (hunk 2 above)
Comment (yakov116) [inline README.md]: Sending me flying up

**Comment by yakov116** [comment]
I can't think of anything to say

**Comment by yakov116** [comment]
Just in case you need one last bump for tonight

**Comment by k0shk0sh** [comment]
@yakov116 with signature
> Sent from my m8 using FastHub

**Comment by k0shk0sh** [comment]
@yakov116 hello
> Sent from my m8 using FastHub

**Comment by k0shk0sh** [comment]
@yakov116 hello

> Sent from my htc m8 using FastHub

**Comment by k0shk0sh** [comment]
Hello work
> Sent from my htc m8 using FastHub

**Comment by yakov116** [comment]
@k0shk0sh not its good?

_Sent from my HTC Nexus 9 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by yakov116** [comment]
Test

_Sent from my Unknown Android SDK built for x86 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

File: README.md [hunk 3]
```diff
@@ -101,7 +101,13 @@ You love FastHub? You want new features or bug fixes?
 Please **contribute** to the  project either by [_creating a PR_](https://github.com/k0shk0sh/FastHub/compare) or [_submitting an issue_](https://github.com/k0shk0sh/FastHub/issues/new) on GitHub.  
 Read the [**contribution guide**](.github/CONTRIBUTING.md) for more detailed information.
 
-- [Translation contributors list](.github/language_contributors.md)
+### [Language Contributors](.github/language_contributors.md)
+
+- Turkish [@kutsan](https://github.com/kutsan)
+- German [@failex234](https://github.com/failex234)
```
Comment (k0shk0sh) [inline README.md]: hello this is a preview will be deleted from phone.

Comment (k0shk0sh) [inline README.md]: hello review.

File: README.md [hunk 4]
```diff
@@ -101,7 +101,13 @@ You love FastHub? You want new features or bug fixes?
...
+- Chinese [@Devifish](https://github.com/Devifish)
+- Indonesian [@dikiaap](https://github.com/dikiaap)
+- Russian [@dedepete](https://github.com/dedepete)
 
 # License
```
Comment (k0shk0sh) [inline README.md]: hello review.

Comment (k0shk0sh) [inline README.md]: hello review 2

Comment (k0shk0sh) [inline README.md]: Hello replying to review comment from phone

_Sent from my Htc m8 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

Comment (k0shk0sh) [inline README.md]: Hello review 3

_Sent from my Htc m8 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by k0shk0sh** [comment]
Testing comment

_Sent from my Htc m8 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

File: README.md [hunk 5]
```diff
@@ -101,7 +101,13 @@ You love FastHub? You want new features or bug fixes?
 Please **contribute** to the  project either by [_creating a PR_](https://github.com/k0shk0sh/FastHub/compare) or [_submitting an issue_](https://github.com/k0shk0sh/FastHub/issues/new) on GitHub.  
 Read the [**contribution guide**](.github/CONTRIBUTING.md) for more detailed information.
 
-- [Translation contributors list](.github/language_contributors.md)
+### [Language Contributors](.github/language_contributors.md)
+
+- Turkish [@kutsan](https://github.com/kutsan)
+- German [@failex234](https://github.com/failex234)
+- Chinese [@Devifish](https://github.com/Devifish)
```
Comment (k0shk0sh) [inline README.md]: testing

Comment (k0shk0sh) [inline README.md]: Testing

_Sent from my Htc m8 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

File: README.md [hunk 6]
```diff
@@ -101,7 +101,13 @@ You love FastHub? You want new features or bug fixes?
...
+- Indonesian [@dikiaap](https://github.com/dikiaap)
+- Russian [@dedepete](https://github.com/dedepete)
```
Comment (k0shk0sh) [inline README.md]: can you add Japanese translation in here & also in the /.github/language_contributors.md ?

**Comment by yakov116** [comment]
@k0shk0sh sure

File: README.md (hunk 6 above)
Comment (yakov116) [inline README.md]: Test sub comment

**Pull request closed without merging**